client/js/actions/server-action-creator.js

```diff
@@ -25,6 +25,12 @@ var ServerActionCreator = {
       actionType: AppConstants.ServerActionTypes.FETCHED_MONTH_DATA,
       monthData: monthData
     });
+  },
+  receiveAvailableEmployees: function(data){
```

Comment:
Could you change this parameter to be called employeeData? All the other methods on the server action creator follow that style, e.g. dayData, weekData